test/unit/containers/solsandbounds.jl

```diff
@@ -220,7 +292,25 @@ function solution_unit()
     primal_sol = Solution(model, soldecs, solvals, 12.3, Coluna.ColunaBase.FEASIBLE_SOL)
     test_solution_iterations(primal_sol, dict_sol)
     @test Coluna.ColunaBase.getvalue(primal_sol) == 12.3
-    #Coluna.ColunaBase.setvalue!(primal_sol, 123.4)
-    #@test Coluna.ColunaBase.getvalue(primal_sol) == 123.4
+    @test Coluna.ColunaBase.getstatus(primal_sol) == Coluna.ColunaBase.FEASIBLE_SOL
+    
+    dict_sol = Dict(1 => 2.0, 2 => 3.0, 3 => 4.0)
+    soldecs = [1,2,3]
+    solvals = [2.0,3.0,4.0]
+
+    primal_sol = Solution(model, soldecs, solvals, 0.0, Coluna.ColunaBase.FEASIBLE_SOL)
+    
+    @test iterate(primal_sol) == iterate(primal_sol.sol)
+    ((_,_), state) = iterate(primal_sol)
```

Comment:
```suggestion
    _, state = iterate(primal_sol)
```
the first var is a pair and you don't use it

Reply:
ok